CMakeLists.txt

```diff
@@ -177,6 +177,7 @@ set (HEADERS
    ${SOURCE_DIR}/src/mongoc/mongoc-opcode.h
    ${SOURCE_DIR}/src/mongoc/mongoc-opcode-private.h
    ${SOURCE_DIR}/src/mongoc/mongoc-read-concern.h
+   ${SOURCE_DIR}/src/mongoc/mongoc-read-concern-private.h
```

Comment:
Please note that the C++11 driver no longer requires this private header. I recommend against installing it.